internal/restorer/progressprinter/progressprinter.go

```diff
@@ -0,0 +1,113 @@
+package progressprinter
```

Comment:
Please move the code to `internal/ui/restore`. I'd like to keep the ui code in one place.